src/test/osd/osd-scrub-snaps.sh

```diff
@@ -151,16 +151,14 @@ function TEST_scrub_snaps() {
     wait_for_clean || return 1
 
     sleep 5
-    ceph pg scrub ${poolid}.0
-    timeout 30 ceph -w
+    pg_scrub "${poolid}.0"
+    timeout 1 ceph -w
```

Comment:
@dachary Because pg_scrub waits for completion of the scrub I don't think ceph -w will output any of the log messages that occurred during the scrub.  It would be better to have a NOWAIT option to pg_scrub.  This code would have to background the ceph -w then wait for the scrub to finish and finally kill the background job and continue.